src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/OptimizedInboxTextEncoder.cs

```diff
@@ -9,6 +9,10 @@
 using System.Runtime.Intrinsics.X86;
 #endif
 
+#if NET5_0_OR_GREATER
```

Comment:
>NET5_0_OR_GREATER [](start = 4, length = 17)

where is this defined? I am not seeing it in the proj file.

Reply:
They are part of the SDK: https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md